spec/index.bs

```diff
@@ -548,7 +548,7 @@ We define an Email schema as the basis for a credential.
 
 <pre class="example" highlight="json">
 {
-  "type": "https://credentials.workday.com/docs/credential-schema.json",
+  "type": "https://credentials.workday.com/docs/specification/v1.0/schema.json",
```

Comment:
We should point this to the current repo until the spec stabilizes. Also this will probably cause issues for non workday consumers of this url:

```
"pattern": "^did:work:[1-9A-HJ-NP-Za-km-z]{16,};id=[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12};version=\\d+.\\d+$"
```

Reply:
My PR assumed it would be hosted somewhere like: https://w3c-ccg.github.io/vc-json-schemas/schema/credential-schema.json for now

Reply:
yes, you're right. I will amend

Reply:
I can add it there if it helps

Reply:
added